app/kumactl/cmd/uninstall/subcommands.go

```diff
@@ -0,0 +1,9 @@
+// +build !windows
+
+package uninstall
+
+import "github.com/spf13/cobra"
+
+var subcommands = []func() *cobra.Command{
+	newUninstallTransparentProxy,
```

Comment:
Rather than making these commands vanish on Windows, I'd suggest making a Windows-specific version of `newUninstallTransparentProxy` and `newInstallTransparentProxy` that just error out with an "unsupported on this platform" message.

That way, we keep a consistent CLI interface, users can be deterministically aware of the features that aren't available, and we keep the build machinery simpler.

Reply:
done